gravitee-apim-console-webui/src/management/api/creation-v4/models/ApiCreationPayload.ts

```diff
@@ -29,6 +29,8 @@ export type ApiCreationPayload = Partial<{
     name: string;
     icon: string;
     supportedListenerType: ListenerType;
+    supportedQos?: Qos[];
```

Comment:
Question: Do we need to `supportedQos` in the API creation payload ? Why don't we use just `selectedQos` ?

Reply:
The supportedQos is the list of Qos that a given entrypoint can handle. It is needed to fill the select options.